src/main/java/hudson/plugins/jira/listissuesparameter/JiraIssueParameterDefinition.java

```diff
@@ -110,9 +120,19 @@ public String getDisplayName() {
         public final String key;
         public final String summary;
 
-        public Result(final Issue issue) {
+        public Result(final Issue issue, String altSummaryFields) {
             this.key = issue.getKey();
-            this.summary = issue.getSummary();
+            if(altSummaryFields == null || altSummaryFields.isEmpty() ) {
```

Comment:
pls use `StringUtils` for such checks, see https://github.com/jenkinsci/jira-plugin/blob/master/src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java#L75

Reply:
Done